sound/soc/sof/sof-client.c

```diff
@@ -0,0 +1,109 @@
+// SPDX-License-Identifier: GPL-2.0-only
+/*
+ * Copyright(c) 2021 Intel Corporation. All rights reserved.
+ * Author: Ranjani Sridharan <ranjani.sridharan@linux.intel.com>
+ */
```

Comment:
Mark Brown  says "The whole block in C++ style please"

we'll have to adjust older files but let's follow his convention.

Reply:
OK, If Mark said, then let's do that. Other subsystems require this style....